nautobot_floor_plan/templates/nautobot_floor_plan/floorplan_retrieve.html

```diff
@@ -62,6 +62,10 @@
                 <td>Tiles</td>
                 <td><a href="{% url 'plugins:nautobot_floor_plan:floorplantile_list' %}?floor_plan={{ object.pk }}">{{ object.tiles.count }}</a></td>
             </tr>
+            <tr>
+                <td>Child Floor Plans</td>
+                <td><a href="{% url 'dcim:location' object.location.pk %}">{{ object.location|count_children_floor_plans }}</a></td>
```

Comment:
Can we change this href to the `Child Floor Plan(s)` tab instead?

Reply:
I played around with this initially, but had issues with highlighting the proper active tab.

Reply:
In that case, let's see if we can instead send it to the Floor Plan List View with a filter pre-applied to only show floor plans for locations that have the source location as a parent.

Reply:
Created a filter and updated the href to the below code. 

```html
<td><a href="{% url 'plugins:nautobot_floor_plan:floorplan_list'%}?parent_location={{ object.location.pk }}">{{ object.location|count_children_floor_plans }}</a></td>